src/main/java/org/killbill/billing/plugin/avatax/api/AvaTaxTaxCalculator.java

```diff
@@ -226,7 +227,7 @@ private GetTaxRequest toTaxRequest(final String companyCode,
         taxRequest.ExemptionNo = null;
         taxRequest.Discount = BigDecimal.ZERO;
         // Required for VAT
-        taxRequest.BusinessIdentificationNo = null;
+        taxRequest.BusinessIdentificationNo = PluginProperties.findPluginPropertyValue(BUSINESS_IDENTIFICATION_NUMBER, pluginProperties);
```

Comment:
Could you add an integration test that specifies it (see `TestAvaTaxTaxCalculator` for instance)? Not sure if we can verify that Avalara correctly takes it into account (maybe there's something in the response?), but at least it would verify the requests still go through.